CesiumGltf/include/CesiumGltf/PropertyView.h

```diff
@@ -381,6 +459,68 @@ template <typename ElementType> class PropertyView<ElementType, false> {
     }
   }
 
+  /**
+   * @brief Constructs a property instance from a class definition and enum
+   * definition.
+   */
+  PropertyView(
```

Comment:
I kinda feel the same way about the duplication here. I think allowing `pEnumDefinition = nullptr` and doing an extra branch for enums would reduce duplication. Assuming everything else can maintain correctness.